src/components/icon/icon.stories.tsx

```diff
@@ -0,0 +1,278 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import React from "react"
+
+import { Icon } from "./icon"
+
+const meta = {
+  component: Icon,
+  tags: ["autodocs"],
+  title: "Components/Icon"
```

Comment:
Please change this line `title: "01. Design System/Icons"`

Reply:
fixed